cmd/internal/cli/action_flags.go

```diff
@@ -229,7 +229,7 @@ func initBoolVars() {
 	actionFlags.SetAnnotation("nohttps", "envkey", []string{"NOHTTPS"})
 
 	// --docker-login
-	actionFlags.BoolVar(&dockerLogin, "docker-login", false, "interactive prompt for docker authentication")
+	actionFlags.BoolVar(&dockerLogin, "docker-login", false, "Login to a Docker Repository interactively")
```

Comment:
Shouldn't it start with a lowercase letter for consistency as everywhere else?

Reply:
Fixed